ci/run_tests.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 The CapableVMs "CHERI Examples" Contributors.
+# SPDX-License-Identifier: MIT OR Apache-2.0
+#
+# Test that examples execute as expected.
+#
+# This is designed to be run by .buildbot.sh, but can also be run manually too,
+# as long as you have a suitable environment, and an execution platform set up
+# and running so that the make-based run-* targets have something to connect to.
+# Notable environment variables:
+#
+# - BUILDBOT_PLATFORM
```

Comment:
This comment is related to `cheri-examples`. Update it to match your case.

Reply:
Thanks. corrected in [2cc9b31d](https://github.com/capablevms/bdwgc/pull/11/commits/2cc9b31d999e78ceced82eadae95bcd57306d72a).